Danger/Assets/Scripts/Classes/Territory.cs

```diff
@@ -3,21 +3,44 @@
 using System.Collections.Generic;
 using UnityEngine;
 using System.Linq;
+using UnityEngine.UI;
 
 public class Territory {
 
     public readonly string Name;
     public readonly Continent Continent;
     public int ID;
     public List<Territory> WhereCanMove = new List<Territory>();
+    public Image Image;
+    public Color ModColor {  get
+        {
+            return Owner.PlayerColor;
+        } }
 
     public Territory(string name, Continent continent)
     {
         Name = name;
         Continent = continent;
     }
 
-    public Player Owner = GameManager.NotOwned; // default is Neutral until it is later claimed
+    public void SetOwner(Player newOwner)
```

Comment:
Function has been added since we need to update the color when a new owner has been set, so this should make it easier to do that